com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md

```diff
@@ -53,47 +53,69 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
 
 **Minimum Depth** set to 2, **Maximum Depth** set to 2: indirect lighting only (1 bounce)
 
-## Materials parameterization
+## Setting path tracing parameters for Materials
+
+
+Path tracing in HDRP makes your scene appear more realistic. To do this, path tracing implements more precise light transport simulations than rasterization. The result is more realistic effects, especially for:
 
-Some light phenomena can be expressed more naturally in path tracing than in rasterization, for instance:
 
 - Light refraction in transmissive objects.
 - Light absorption in transmissive objects.
 - Subsurface scattering.
 
-Rasterization uses various methods to approximate complex lighting effects, with each of those methods relying on dedicated Material parameters. Path tracing computes those same effects all at once, without the need for as many parameters.
+Rasterization uses separate methods to approximate lighting effects, which require multiple Material parameters. Path tracing computes all lighting effects and how light interacts with Materials at the same time, which means Materials in your scene need fewer Material parameters.
```

Comment:
>how light interacts with Materials
we don't need to say tat, just "Path tracing computes all lighting effects at the same time" is enoguh